lightning/src/chain/channelmonitor.rs

```diff
@@ -906,6 +913,10 @@ pub(crate) struct ChannelMonitorImpl<Signer: EcdsaChannelSigner> {
 	// of block connection between ChannelMonitors and the ChannelManager.
 	funding_spend_seen: bool,
 
+	/// True if the commitment transaction fee is paid by us.
+	/// Added in 0.0.124.
+	holder_pays_commitment_tx_fee: Option<bool>,
```

Comment:
Nah, it doesn't change in any way for the commitment tx fees. This still applies:

> Since the fees are currently one-sided (the party which requested the channel creation always pays the fees for the commitment transaction), it's simplest to only allow it to set fee levels; however, as the same fee rate applies to HTLC transactions, the receiving node must also care about the reasonableness of the fee.

https://github.com/lightning/bolts/blob/master/02-peer-protocol.md#rationale-25